src/components/Synthetics/TradeBox/TradeBox.tsx

```diff
@@ -1512,7 +1513,14 @@ export function TradeBox(p: Props) {
 
               <div className="App-card-divider" />
 
-              {feesType && <TradeFeesRow {...fees} executionFee={executionFee} feesType={feesType} />}
+              {feesType && (
+                <TradeFeesRow
+                  {...fees}
+                  executionFee={executionFee}
+                  feesType={feesType}
+                  warning={executionFee?.warning}
```

Comment:
let's remove `warning` prop and use `executionFee?.warning` inside TradeFeesRow  instead?